assets/css/hca.scss

```diff
@@ -236,10 +236,19 @@ button.hca-button-green {
       }
     }
     .edit-btn { 
-      padding: 1.5rem 3.5rem 1.5rem 3rem;
+      padding: 1rem;  // 1.5rem 3.5rem 1.5rem 3rem;
+      border: 2px solid $color-primary;
+      &.primary-outline {
+        background-color: transparent;
+        color: $color-primary;
+        &:hover {
+          background-color: $color-primary;
```

Comment:
I feel like this hover styling is too harsh (turning the background from transparent to that bright blue). Most of our button hover styling is to make the colors slightly darker. It's a bit harder to see for the outline buttons, but it's what we're doing elsewhere. Unless we can come up with a better solution than that, let's stick with that!